Dockerfile-alpine.template

```diff
@@ -42,6 +42,9 @@ RUN set -eux; \
 		$DOCKER_PG_LLVM_DEPS \
 		bison \
 		coreutils \
+{{ if .major >= 17 then ( -}}
+		docbook-xml \
+{{ ) else "" end -}}
```

Comment:
At the risk of invoking Chesterton's Fence, do we recall / can we find whether we actually enabled building of man pages intentionally?  (IMO it'd be pretty reasonable, _especially_ in our Alpine variants, to not do so, especially as we're focused on size more than "meet absolutely every use case" and the documentation is all available online :innocent:)

Reply:
No reasoning for man pages or docs in https://github.com/docker-library/postgres/pull/216 or https://github.com/docker-library/postgres/pull/119. So maybe safe to drop generally, but at least for 17+.

Reply:
This is the PR that introduced Alpine: https://github.com/docker-library/postgres/pull/216. There is no mention of manuals

Reply:
.....we build them and then we delete them.....

https://github.com/docker-library/postgres/blob/master/16/alpine3.20/Dockerfile#L154-L157

Reply:
Is it 1`world-bin` or `all`?

Reply:
`all` worked. Only "issue" preventing us from removing the lines to delete the docs is contrib are still generating them

Reply:
I think we do need `world-bin` and `install-world-bin` in order to get the `contrib/` modules that are included in `world` (and are not part of `all`). And we should just apply it to all versions since we are deleting the docs anyway.

Reply:
I wanted to do it for all but was worried I was bringing scope creep